core/primitives/src/views.rs

```diff
@@ -2935,7 +2935,7 @@ mod tests {
         let config_store = RuntimeConfigStore::new(None);
         let config = config_store.get_config(PROTOCOL_VERSION);
         let view = RuntimeConfigView::from(RuntimeConfig::clone(config));
-        insta::assert_json_snapshot!(&view);
+        insta::assert_json_snapshot!(&view, { ".wasm_config.vm_kind" => "vm_kind"});
```

Comment:
I would use something like this instead to make it clear the value has been tampered with:


```suggestion
        insta::assert_json_snapshot!(&view, { ".wasm_config.vm_kind" => "<REDACTED>"});
```